EventsManagementService/EventManagementService.Domain/Mappers/EventMapper.cs

```diff
@@ -0,0 +1,40 @@
+﻿using CoreEvent = EventManagementService.Domain.Models.JobEvent;
+using DbEvent = EventManagementService.Infrastructure.Persistence.Entities.JobEvent;
+
+namespace EventManagementService.Domain.Mappers
+{
+    public static class EventMapper
+    {
+        public static CoreEvent ToCoreEvent(DbEvent dbEvent)
+        {
+            var coreEvent = new CoreEvent();
+
+            coreEvent.Id = dbEvent.Id;
+            coreEvent.EmployeeId = dbEvent.EmployeeId;
+            coreEvent.PetId = dbEvent.PetId;
+            coreEvent.PetServiceId = dbEvent.PetServiceId;
+            coreEvent.EventDate = dbEvent.EventDate;
+            coreEvent.Completed = dbEvent.Completed;
+
+            coreEvent.Employee = new Models.Employee() { Id = dbEvent.EmployeeId, FirstName = dbEvent.Employee.FirstName, LastName = dbEvent.Employee.LastName, Role = dbEvent.Employee.Role };

```

Comment:
There's a potential for bug here and you've done this before.

What happens in dbEvent.Employee is null?